service/server.go

```diff
@@ -47,6 +49,15 @@ func serve() {
 	router.HandleFunc("/health", Health).Methods("GET")
 	router.HandleFunc("/", Up).Methods("GET", "POST")
 
+	//Creates an LRU cache of the given size
+	var err error
+	db.LruCache, err = lru.New(5)
```

Comment:
@Lucy911, does it make sense to constant-ize the value `5` here? if you do that, you can potentially answer the questions of "why 5?" and "what unit is this?" based on the name of the const

Reply:
same question here, why initial size is 5? Are you only insert one item with key "dbschema" into the cache?

Reply:
@davemasselink Yes, you are correct. The value 5 is the size of Cache items. For now we are using only one item. If later we need more items to cache, we can increase this number.

Reply:
@formuzi Made initial size is 10. Right now only one item which is the whole set of dbschema.